modules/cnv-updating-imported-vmware-vm-network-name.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-vmware-vm-to-ocp.adoc
+[id="cnv-updating-imported-vmware-vm-network-name_{context}"]
+= Updating an imported VMware virtual machine's NIC
+
+Update the imported virtual machine's NIC configuration file to conform to {ProductName} naming conventions.
+
+.Procedure
+
+. Log in to the imported virtual machine.
+. Go to the */etc/sysconfig/network-scripts* directory.
+. Rename the network configuration file:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+$ mv VMnet__1__ ifcfg-_name_
+----
```

Comment:
This part need to get updated to:

Rename the network configuration file:
From VMware interface name, to 'ifcfg-ethX' (For a single nic, normally this would be called "ifcfg-eth0").
In addition, this file content needs to get updated, as follows:
Both network interface NAME and DEVICE, need to change from VMware interface name/device into 'ethX' 
(normally for a single nic should be "eth0").

Reply:
`Both network interface NAME and DEVICE, need to change from VMware interface name/device into 'ethX'`

Not sure what you mean. Normally ifcfg file contains DEVICE, not NAME. I will add step to update DEVICE but you will need to clarify what you mean by NAME in the file.